File: fe_camphub/src/libs/core/constants/BookingStatus.ts
    export enum BookingStatus {
        PENDING_CONFIRM = "PENDING_CONFIRM", // Chờ chủ đồ xác nhận
        PAID_REJECTED = "PAID_REJECTED", // chủ đồ từ chối đơn
        WAITING_DELIVERY = "WAITING_DELIVERY", // chờ giao hàng
        IN_USE = "IN_USE", // đang thuê
        DUE_FOR_RETURN = "DUE_FOR_RETURN", // đến hạn trả
        RETURNED_PENDING_CHECK = "RETURNED_PENDING_CHECK", // đang giao trả
        RETURN_REFUND_REQUESTED = "RETURN_REFUND_REQUESTED", // đang trả hàng/hoàn tiền
        RETURN_REFUND_PROCESSING = "RETURN_REFUND_PROCESSING", // đã nhận hàng trả - chờ admin xử lý
        WAITING_REFUND = "WAITING_REFUND", // chử hoàn tiền
        COMPLETED = "COMPLETED", // hoàn thành
        DISPUTE_PENDING_REVIEW = "DISPUTE_PENDING_REVIEW", // có khiếu nại từ chủ thuê
        COMPENSATION_COMPLETED = "COMPENSATION_COMPLETED", // đã bồi thường
        LATE_RETURN = "LATE_RETURN", // đơn hàng trễ hạn trả
        OVERDUE = "OVERDUE", // đơn hàng quá hạn trả
        DAMAGED_ITEM = "DAMAGED_ITEM", // đồ bị hỏng
        FORFEITED = "FORFEITED", // đồ bị mất
    }

File: be_camphub/src/main/java/org/camphub/be_camphub/enums/BookingStatus.java
package org.camphub.be_camphub.enums;

public enum BookingStatus {
    PENDING_CONFIRM, // đợi chủ thuê xác nhận
    PAID_REJECTED, // đã thanh toán nhưng bị từ chối
    WAITING_DELIVERY, // chờ giao hàng
    IN_USE, // đang sử dụng
    DUE_FOR_RETURN, // đến hạn trả
    RETURNED_PENDING_CHECK, // đã trả, chờ kiểm tra
    RETURN_REFUND_REQUESTED, // đã trả, yêu cầu hoàn tiền
    RETURN_REFUND_PROCESSING, // chủ thuê đã xác nhận trả, chờ admin xử lý hoàn tiền
    WAITING_REFUND, // chờ hoàn tiền
    COMPLETED, // hoàn thành
    DISPUTE_PENDING_REVIEW, // khiếu nại, chờ xử lý
    COMPENSATION_COMPLETED, // bồi thường hoàn thành
    LATE_RETURN, // trả muộn
    OVERDUE, // quá hạn
    DAMAGED_ITEM, // hàng trả bị hư hỏng
    FORFEITED, // mất đồ
}
